src/index.ts

```diff
@@ -1,58 +1,42 @@
 const argv = require('minimist')(process.argv.slice(2));
```

Comment:
I could not figure out what I had to import from minimist to replace this require with an import to make eslint happy...

docs: https://github.com/substack/minimist

Reply:
Going off https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-var-requires.md

It looks like it wants:
```
import minimist = require("minimist");
const argv = minimist(process.argv.slice(2));
```

Reply:
ah makes sense